### PR TITLE
User can comment on a game

### PR DESCRIPTION
**Pull request by c-alvin**
issue3finalprojectmobile

issue3windowsfinal

### CODE REVIEW

**Comment by c-alvin** [comment]
image
image

File: client/pages/details.jsx [hunk 1]
```diff
@@ -1,83 +1,176 @@
...
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(this.state)
+    };
+    fetch('/api/details/comment', req)
+      .then(res => res.json())
+      .then(result => {
+        const test = this.state.comments;
+        test.push(result);
```
Comment (thebearingedge) [inline client/pages/details.jsx:57]: Be careful, this is not a copy. You need to slice the array to make a copy. Assigning an object/array/function to a variable does not copy it. It copies the memory reference to the same object/array/function.

File: server/public/styles.css [hunk 2]
```diff
@@ -113,6 +118,34 @@ body {
...
+  margin-bottom: -0.5rem;
+}
+
+#button-white {
+  color: white;
+}
+
 /* .background-image {
   background-image: url("https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/5b1089c6-bacb-4d88-8a75-3692aff4c6b1/dc68p85-ce4fdf1a-5ef0-4eb7-8e0d-8cd8d0745ea5.png/v1/fill/w_1192,h_670,q_70,strp/hyrule___legend_of_zelda__breath_of_the_wild_by_sephiroth508_dc68p85-pre.jpg?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7ImhlaWdodCI6Ijw9MjE2MCIsInBhdGgiOiJcL2ZcLzViMTA4OWM2LWJhY2ItNGQ4OC04YTc1LTM2OTJhZmY0YzZiMVwvZGM2OHA4NS1jZTRmZGYxYS01ZWYwLTRlYjctOGUwZC04Y2Q4ZDA3NDVlYTUucG5nIiwid2lkdGgiOiI8PTM4NDAifV1dLCJhdWQiOlsidXJuOnNlcnZpY2U6aW1hZ2Uub3BlcmF0aW9ucyJdfQ.K3Way6CnhVC2A7DPj3G-B7tfcpyuLbYnt_MwzdMGIEQ");
   opacity: 0.75;
```
Comment (thebearingedge) [inline server/public/styles.css:151]: Please remove all commented out code.

File: client/pages/details.jsx [hunk 3]
```diff
@@ -1,83 +1,176 @@
...
         </div>
         <div className='col'>
-          <Badge id="rating" bg="info">{rating}</Badge>
+            <Badge id="rating" bg="info">{rating}</Badge>
         </div>
       </div>
       <div className='row'>
         <div className='color-text-white col-md-6 font-lig'>
           <h1 className='color-text-lightblue margin-top-small font-lig font-size-large'>SUMMARY</h1>
+            {/* <img src="https://img.youtube.com/vi/L93H7YC-83o/hqdefault.jpg " alt="" /> */}
```
Comment (thebearingedge) [inline client/pages/details.jsx:126]: Please remove all commented out code.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**